templates/org/aac/aac-topic-landing-page.md

```diff
@@ -1,42 +1,43 @@
 ---
-title: TOPIC NAME on Azure
-description: A guide to building running TOPIC NAME on Azure
-author: 
-ms.date: 2/4/2019
+title: <Topic name> on Azure
+description: A guide to building running <Topic name> on Azure
+author: <Contributor's GitHub username>
+ms.author: <Contributor's Microsoft alias. This can include multiple contributors, separated by commas.> 
+ms.date: <Date of publish or major update, in mm/dd/yyyy format>
+ms.topic: landing-page
+ms.service: azure-architecture-center
 ---
 <!-- markdownlint-disable MD033 -->
 <!-- markdownlint-disable MD026 -->
 
-# TOPIC NAME
+# <!--Topic name-->
 
-## Introduction to TOPIC
+## Introduction to <!--Topic-->
```

Comment:
```suggestion
```
Deleting this because it won't pass PR review and is against Microsoft style. An H2 shouldn't be stacked directly under the H1.